### DISCUSSION TITLE
border-1 should exist

### DISCUSSION THREAD

**Discussion by ayadgu** [tailwindlabs/tailwindcss#5787] [Ideas]
I know `border` exists already but maybe other people too were puzzled when not getting any results with `border-1`.

This same idea should be considered for all classes with the same pattern.

**Comment by NovusOpus** [comment]
To be honest, Right now, I don't think it's necessary, Specially now with [JIT Arbitrary Value Support](https://tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support), you already  can just do: `border-[1px]`, `border-[1rem]`, `border-[1.5rem]` or any custom **valid** value.

Sure you could say it's not as clean as just: `border-1` but again, `border` already does the same and is more cleaner, apart from the fact that most people is already familiar with this way.

So having people from Tailwind CSS Adding a bunch of other classes(**That Already Exist**) just don't make much sense, Specially now with Tailwind CSS V3 Launch around the corner, and they also working on updating and improving the docs so it's ready for V3 and also making it more easy and clearer to understand than it was.

But maybe at some point in the future they could just add those classes, and who knows maybe they are coming with V3?
